commands/rootCmd.go

```diff
@@ -0,0 +1,83 @@
+/*
+Package commands contains github-audit shell commands related logic.
+*/
+package commands
+
+import (
+	"context"
+	"os"
+
+	"github.com/maplelabs/github-audit/input"
+	"github.com/maplelabs/github-audit/internal/configurator"
+	"github.com/maplelabs/github-audit/internal/taskmanager"
+	"github.com/maplelabs/github-audit/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//ConfigFile stores config.yaml location
+	ConfigFile string
+	log        logger.Logger
+)
+
+// rootCmd represents the base command which is executed in normal run of github-audit.
+var rootCmd = &cobra.Command{
+	Use:   "github-audit",
+	Short: "github-audit monitors github repositories",
+	Long: `github-audit monitors github commits , operations and sends to observability platform.
+To run github-audit use subcommand "start" 
+Ex: github-audit start --config=<path to config.yaml>
+
+To stop github-audit use subcommand "stop"
+Ex: github-audit stop
+	`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// package main ctx received here
+		ctx := cmd.Context()
+		return StartGitAudit(ctx)
+	},
+}
+
+// Execute is the starting point to rootCmd.
+func Execute(ctx context.Context) {
+	// disabled default completion options.
+	rootCmd.CompletionOptions.DisableDefaultCmd = true
+	if err := rootCmd.ExecuteContext(ctx); err != nil {
+		log.Errorf("error[%v] in root command execution", err)
+		os.Exit(0)
```

Comment:
exit code should not 0 for failures like config validation you may keep as 1, and choose other positive number for other type of failures.